import/react-native-image-picker/Fable.Import.ReactNativeImagePicker.fs

```diff
@@ -0,0 +1,67 @@
+namespace Fable.Import
+
+open System
+open Fable.Core
+open Fable.Import
+open Fable.Import.JS
+open Fable.Import.Browser
+
+module ReactImagePicker =
+
+    [<StringEnum>]
+    type CameraType =
+    | [<CompiledName("front")>] Front
```

Comment:
By default, the first letter is lowered for `StringEnum` cases so you only need to use `CompiledName` if you actually **don't want** the first letter lowered (or you want to use a completely different name than the case name).